src/sigal/plugins/lrp.py

```diff
@@ -0,0 +1,47 @@
+"""Plugin which add raw support.
+
+Plugin make temp directory which is use for generate jpeg from raw file (jpeg is camera-generated thumbnail).
+For this dcraw program is used.
+
+"""
+
+import logging
+
+from sigal import signals
+
+import os
+import sys
+import re
+
+from ..gallery import Image
+from ..settings import Status, get_thumb
+import copy
+
+logger = logging.getLogger(__name__)
+
+tempDir="/tmp/rawTempSigal"
```

Comment:
This isn't Windows compatible

Reply:
Use `tempfile.gettempdir` instead: https://docs.python.org/3.12/library/tempfile.html#tempfile.gettempdir